webui/react/src/pages/ExperimentDetails/ExperimentActions.tsx

```diff
@@ -52,16 +53,26 @@ const ExperimentActions: React.FC<Props> = ({ experiment, onClick, onSettled }:
     Tensorboard: false,
   });
 
-  const handleArchive = useCallback((archive: boolean) => async (): Promise<void> => {
+  const handleArchive = useCallback(() => async (): Promise<void> => {
     setBtnLoadingStates(state => ({ ...state, Archive: true }));
     try {
-      await archiveExperiment(experiment.id, archive);
+      await archiveExperiment({ experimentId: experiment.id });
       onSettled();
     } finally {
       setBtnLoadingStates(state => ({ ...state, Archive: false }));
     }
   }, [ experiment.id, onSettled ]);
 
+  const handleUnarchive = useCallback(() => async (): Promise<void> => {
+    setBtnLoadingStates(state => ({ ...state, archive: false }));
```

Comment:
I think I landed a change today here. short version is this line doesn't work  (It was an existing issue before this PR) if you rebase it should be fixed.

Reply:
That should be fixed too.